sirepo/pkcli/job_agent.py

```diff
@@ -208,6 +213,14 @@ def terminate(self):
         finally:
             tornado.ioloop.IOLoop.current().stop()
 
+    def _get_cmd_type(self, msg):
+        c = _Cmd
+        if msg.jobRunMode == job.SBATCH:
+            c = _SbatchRun if msg.isParallel else _SbatchCmd
```

Comment:
`return`, don't assign, then the code is cleaner:

```py
if msg.jobRunMode == job.SBATCH:
     return _SbatchRun if msg.isParallel else _SbatchCmd
elif msg.jobCmd == 'fastcgi':
     return _FastCgiCmd
return _Cmd
```

Easier to follow the logic, especially for the default case